setup.py

```diff
@@ -0,0 +1,27 @@
+import pathlib
+
+from setuptools import setup
+
+HERE = pathlib.Path(__file__).parent
+README = (HERE / "README.md").read_text()
+
+setup(
+    name="peerdid",
+    version="0.1.0",
+    description="PeerDID for Python",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/sicpa-dlab/peer-did-python",
+    author="",
+    author_email="",
+    license="Apache-2.0",
+    python_requires=">=3.7",
```

Comment:
Is there any feature of 3.7 that we're depending on? Or could 3.5 suffice? (I don't really have a problem with 3.7, but I think we should declare the earliest version of Python that is adequate for our needs. If that's 3.7, fine. Just checking.)

Reply:
There is no problem in changing the python version to 3.5. I will change it in the next PR